src/main.ts

```diff
@@ -361,9 +374,20 @@ export const processIntent = async (intent: Intent) => {
           "x-api-key": process.env.ORCHESTRATOR_API_KEY!,
           Authorization: `Bearer ${BEARER_TOKEN}`,
         },
+        validateStatus: () => true
```

Comment:
Added this so the request doesn't simply throw with all the extra network object info, handling is done more manually below, can revert if preferred way is to keep throwing on statusCode !== 200

Reply:
when running this in the ci it would still be good to fail bc otherwise even if the reponses fail the ci might pass